lib/config/contributors.js

```diff
@@ -64,7 +64,17 @@ async function loadContributorFromConfiguration(author) {
     path.resolve(CONTRIBUTORS_PICTURES_LOCATION, author + '.jpg'),
   );
 
-  authorInformation.picture = hasPictureFile ? author : 'no-photo';
+  if (authorInformation.picture !== 'no-photo') {
+    if (!hasPictureFile) {
+      throw new Error(
+        `Could not find author picture for "${author}".` +
```

Comment:
Maybe it should also say "Did you use .jpeg instead of .jpg"? That seems to be what happened in the previous issue.

Alternatively, it could be tolerant and accept either.

Reply:
Done